python/lsst/faro/utils/extinction_corr.py

```diff
@@ -37,7 +37,45 @@
         e.msg,
     )
 
-__all__ = ("extinction_corr",)
+__all__ = ("extinction_corr", "extinctionCorrTable",)
+
+
+def extinctionCorrTable(coords, bands):
+
+    # Extinction coefficients for HSC filters for conversion from E(B-V) to extinction, A_filter.
+    # Numbers provided by Masayuki Tanaka (NAOJ).
+    #
+    # Band, A_filter/E(B-V)
+    extinctionCoeffs_HSC = {
```

Comment:
Shouldn't this be a butler dataset or some kind of external configuration? Hard-coding HSC doesn't seem correct to me.

Reply:
Yes, that's an excellent idea. We really should have the [table from Schlafly & Finkbeiner 2011](https://iopscience.iop.org/article/10.1088/0004-637X/737/2/103#apj398709t6) as a lookup table (dict) in the butler. (Supplemented with HSC coefficients, and ultimately ComCam/LSSTCam as they are measured.)

I'm not sure the best way to key that dict -- are the `instrument` and `physical_filter` dimensions sufficient to specify the coefficient lookup?

This work should definitely happen on a separate ticket, though.

Reply:
Yes, I'm sure a separate ticket is fine but we have to make sure it happens. I don't know if @TallJimbo has a plan for this sort of thing as a butler dataset?

Reply:
I do not have any existing plans for this sort of thing; please feel free to propose something and bring any middleware-adjacent design questions you have to the middleware support channel.

Reply:
Is this affected by Yusra's new extinction coefficents in objectTables ticket?

Reply:
Yes, I'm pretty sure these are the same coefficients that @yalsayyad just starting putting in various parquet datasets (so we only end up evaluating the map once per object), so if we're using those datasets as input here, we've got a solution now.

Reply:
I think it is fixed in the analysis_drp version and that we can make this simpler by importing that version for now (with any updates needed) and then long term moving it to a separate package that anything can import from.

Reply:
Future plan is DM-34061. Standard bandpasses are stored in the FGCM lookup table before processing as a CALIB. Intermediate plan is reading from obs_packages: DM-33933